### PR TITLE
Add storageservice.GetStorageServiceKeys

### PR DESCRIPTION
**Pull request by paulmey**

### CODE REVIEW

File: management/storageservice/client.go [hunk 1]
```diff
@@ -83,6 +83,26 @@ func (self StorageServiceClient) GetStorageServiceByLocation(location string) (*
 	return nil, nil
 }
 
+func (self StorageServiceClient) GetStorageServiceKeys(serviceName string) (primary, secondary string, err error) {
```
Comment (ahmetalpbalkan) [inline management/storageservice/client.go]: I would favor an implementation like:

```go
type GetStorageServiceKeysResponse struct { PrimaryKey, SecondaryKey string } 
```

and returning that form here.

Comment (paulmey) [inline management/storageservice/client.go]: :+1:

File: management/storageservice/client.go [hunk 2]
```diff
@@ -83,6 +83,26 @@ func (self StorageServiceClient) GetStorageServiceByLocation(location string) (*
...
+
+	requestURL := fmt.Sprintf(azureStorageServiceURL, serviceName)
+	data, err := self.client.SendAzureGetRequest(requestURL)
+	if err != nil {
+		return GetStorageServiceKeysResponse{}, err
+	}
+
+	response := GetStorageServiceKeysResponse{}
+	err = xml.Unmarshal(data, response)
+	if err != nil {
```
Comment (ahmetalpbalkan) [inline management/storageservice/client.go]: if you look closely this `if` is unnecessary

**Comment by ahmetalpbalkan** [comment]
LGTM

**Pull request merged**